Core/include/Acts/Surfaces/Surface.hpp

```diff
@@ -389,6 +389,8 @@ class Surface : public virtual GeometryObject,
   virtual FreeToPathMatrix freeToPathDerivative(
       const GeometryContext& gctx, const FreeVector& parameters) const;
 
+  virtual BoundVector correctSign(const BoundVector& boundParams) const;
```

Comment:
I am not too happy with this name. Maybe
```suggestion
  virtual BoundVector boundToMeasurement(const BoundVector& boundParams) const;
```
?

Reply:
On thing we need to be careful here is that in principle we do not want to have separate coordinate systems between surface and measurement. As we discussed, the sign in the line surface is an edge case. 

I think we should really avoid introducing the concept of a more or less generic coordinate transform between surface frame and measurement frame.

Reply:
hm makes sense but in principle this is what we do here right?

something else: do we have to propagate that `abs` into the jacobian?

Reply:
> hm makes sense but in principle this is what we do here right?

Basically, I think that having this be an interface method promotes this as a concept.

> something else: do we have to propagate that `abs` into the jacobian?
Now that's a good question. It's not obvious to me if the covariance also contains information on the radius sign.

Reply:
> It's not obvious to me if the covariance also contains information on the radius sign.

I don't think we need to do this for any jacobians during propagation
But I guess the off diagonal elements of <2x2> measurement in KF update should be flipped with drift distance (L)
```
L -> -L
cov <L ,z> -> - cov<L,z>
```